app/pages/sections/service-recipients/template.test.js

```diff
@@ -47,21 +66,61 @@ describe('service-recipients page', () => {
     });
   }));
 
-  it('should render the organisation heading', componentTester(setup, (harness) => {
-    harness.request(context, ($) => {
-      const heading = $('[data-test-id="organisation-heading"]');
-      expect(heading.length).toEqual(1);
-      expect(heading.text().trim()).toEqual(context.organisationHeading);
-    });
-  }));
+  describe('table', () => {
+    it('should render the organisation heading', componentTester(setup, (harness) => {
+      harness.request(context, ($) => {
+        const heading = $('[data-test-id="organisation-heading"]');
+        expect(heading.length).toEqual(1);
+        expect(heading.text().trim()).toEqual(context.organisationHeading);
+      });
+    }));
 
-  it('should render the ods code heading', componentTester(setup, (harness) => {
-    harness.request(context, ($) => {
-      const heading = $('[data-test-id="ods-code-heading"]');
-      expect(heading.length).toEqual(1);
-      expect(heading.text().trim()).toEqual(context.odsCodeHeading);
-    });
-  }));
+    it('should render a checkbox for each organisation name', componentTester(setup, (harness) => {
+      harness.request(context, ($) => {
+        const checkbox1 = $('[data-test-id="organisation-name-checkbox-ods1"]');
+        expect(checkbox1.length).toEqual(1);
+        expect(checkbox1.text().trim()).toEqual(context.tableData[0].organisationName.text);
+
+        const checkboxInput1 = checkbox1.find('input');
+        expect(checkboxInput1.attr('id')).toEqual(context.tableData[0].organisationName.id);
+        expect(checkboxInput1.attr('name')).toEqual(context.tableData[0].organisationName.name);
+        expect(checkboxInput1.attr('type')).toEqual('checkbox');
+        expect(checkboxInput1.attr('checked')).toEqual(undefined);
```

Comment:
checked is `undefined` if not there and has a value of `''` (empty string) if it is there